backend/shared/shared/oidc/auth.py

```diff
@@ -165,4 +165,9 @@ def authenticate(self, request):
         ):
             return None
 
+        # Store organization roles in session
+        from shared.oidc.utils import request_organization_roles
+
+        request_organization_roles(user.oidc_profile.eauthorization_profile, request)
```

Comment:
Should we also clear the session when user log out?

Reply:
I think the session is flushed when user logs out, correct?

Reply:
Ah, true 👍 my bad, I thought the backchannel log out is using a different method to log out